src/Ledger/Gov/Properties.agda

```diff
@@ -1,21 +1,29 @@
 {-# OPTIONS --safe #-}
 
-import Data.List.Membership.Propositional as P
-open import Data.List.Membership.Propositional.Properties
-open import Data.List.Relation.Unary.Any hiding (map)
-
-open import Ledger.Prelude hiding (Any; any?)
```

Comment:
I'm in favor of moving these imports inside the module.

(although, this mixes a no-op/cosmetic mod into the PR... I'm fine with that, but have been told not to do it 😅 )